Apps/Sandcastle/gallery/development/3D Models Instancing.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Create 3D models using glTF.">
+    <meta name="cesium-sandcastle-labels" content="Development">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    "use strict";
+//Sandcastle_Begin
+
+var debugShowBoundingVolume = false;
+var debugWireframe = false;
+var globeVisible = true;
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    globe : globeVisible ? undefined : false,
+    skyAtmosphere : globeVisible ? undefined : false
+});
+
+var scene = viewer.scene;
+var context = scene.context;
+var camera = viewer.camera;
+scene.debugShowFramesPerSecond = true;
+
+var instancedArraysExtension = context._instancedArrays;
+var count = 1024;
+var spacing = 0.0002;
+var url = '../../SampleData/models/CesiumAir/Cesium_Air.bgltf';
+var dynamic = false;
+var useCollection = true;
+
+var centerLongitude = -123.0744619;
+var centerLatitude = 44.0503706;
+var height = 5000.0;
+
+function orientCamera(center, radius) {
+    var controller = scene.screenSpaceCameraController;
+    var r = Math.max(radius, camera.frustum.near);
+    controller.minimumZoomDistance = r * 0.5;
+
+    var heading = Cesium.Math.toRadians(230.0);
+    var pitch = Cesium.Math.toRadians(-20.0);
+    camera.lookAt(center, new Cesium.HeadingPitchRange(heading, pitch, r * 2.0));
+}
+
+function createCollection(instances) {
+    var collection = scene.primitives.add(new Cesium.ModelInstanceCollection({
+        url : url,
+        instances : instances,
+        dynamic : dynamic,
+        debugShowBoundingVolume : debugShowBoundingVolume,
+        debugWireframe : debugWireframe
+    }));
+
+    collection.readyPromise.then(function(collection) {
+        // Play and loop all animations at half-speed
+        collection._model.activeAnimations.addAll({
```

Comment:
I know `ModelInstanceCollection` is private for now, but perhaps it should have a `getter` for `activeAnimations`?